app/assets/javascripts/hubstats/deploys.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function() {
```

Comment:
I don't think you need to copy the code from pull_requests.js.  I believe that all the Javascript from `app/assets/javascripts/hubstats` gets concatenated and included on every page.  If that is the case, you are adding extraneous duplicate event handlers here.